platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/MerkleStateRoot.java

```diff
@@ -16,7 +16,7 @@
 
 package com.swirlds.platform.state;
 
-import static com.swirlds.logging.legacy.LogMarker.EXCEPTION;
+import static com.swirlds.logging.legacy.LogMarker.*;
```

Comment:
Explicit enumeration is preferred over star imports

Reply:
I have a follow up PR that requires this once. Can I fix it there?

Reply:
I mean this PR https://github.com/hashgraph/hedera-services/pull/16477

Reply:
Yep sounds good, thanks!